homeworks/rostyslav.havuka_RostyslavHavuka/6-typeScript/src/index.ts

```diff
@@ -0,0 +1,232 @@
+
+class Game {
+  private currentKey: string = '';
+  private score: number = 100;
+  private intervalId: NodeJS.Timeout;
+  private intervalStatusBarId: NodeJS.Timeout;
+  private gameState: boolean = false;
+  private timerProgressBar: number = 0;
+  private squareWidth: number;
+
+  constructor(
+    private scoreElement = document.querySelector('[data-score]') as HTMLHeadingElement,
```

Comment:
Return style, please 😄 
Create separate variables and pass them to the custructor